### PR TITLE
Change imports for simple markdown

### PR DESCRIPTION
**Pull request by nixterrimus**
## Summary:
Lands after moving simple-markdown into the mono-repo.

Changes the imports to use the new @khanacademy prefix for the package.

Issue: https://khanacademy.atlassian.net/browse/LP-11429

## Test plan:
- Build the repo
- **All should be well**

### CODE REVIEW

**Review by kevinbarabash — COMMENTED** [review]
Thanks for splitting this out into its own PR.  You'll need to update the following files so that these imports work in dev:
- .flowconfig
- .eslintrc.js
- config/test/test.config.js

**Comment by nixterrimus** [comment]
ESLint looks like it handles it already:

```
const pkgAliases = fs
    .readdirSync(path.join(__dirname, "packages"))
    .map((pkgName) => {
        return [
            `@khanacademy/${pkgName}`,
            `./packages/${pkgName}/src/index.js`,
        ];
    });
```

This should resolve simple markdown fine

**Comment by nixterrimus** [comment]
I updated the other two

**Comment by kevinbarabash** [comment]
> ESLint looks like it handles it already:

I forgot what I did yesterday. 🙃

**Review by kevinbarabash — APPROVED** [review]
LGTM.  Thanks for updating the imports.  There's a bit of lint that needs fixing.  🚢

**Comment by nixterrimus** [comment]
@kevinbarabash I know those days...

File: .flowconfig [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.file_ext=.less
 # stub out less imports.
 module.name_mapper.extension='less' -> 'empty/object'
 module.name_mapper='@khanacademy/\(perseus[^/]*\)' -> '<PROJECT_ROOT>/packages/\1/src/index.js'
+module.name_mapper='@khanacademy/simple-markdown)' -> '<PROJECT_ROOT>/packages/simple-markdown/src/index.js'
```
Comment (kevinbarabash) [inline .flowconfig:14]: There's a trailing `)` on the "from" that needs removing.

File: config/test/test.config.js [hunk 2]
```diff
@@ -38,6 +38,8 @@ module.exports = {
     moduleNameMapper: {
         "^@khanacademy/perseus(.*)$":
             "<rootDir>/packages/perseus$1/src/index.js",
+        "^@khanacademy/simple-markdown":
+            "<rootDir>/packages/simple-markdown/src/index.js",
```
Comment (kevinbarabash) [inline config/test/test.config.js:42]: nit: adding a `$` at the end would be more precise

**Pull request merged**